python/lsst/afw/image/exposure/exposure.cc

```diff
@@ -53,126 +54,132 @@ void declareCastConstructor(PyExposure<ToPixelT> &cls) {
+            cls.def("setPhotoCalib", &ExposureT::setPhotoCalib, "photoCalib"_a);
+            cls.def("getPsf", (std::shared_ptr<detection::Psf>(ExposureT::*)()) &ExposureT::getPsf);
+            cls.def("setPsf", &ExposureT::setPsf, "psf"_a);
+            cls.def("hasPsf", &ExposureT::hasPsf);
+            cls.def("getInfo", (std::shared_ptr<ExposureInfo>(ExposureT::*)()) &ExposureT::getInfo);
+            cls.def("setInfo", &ExposureT::setInfo, "exposureInfo"_a);
+
+            cls.def("subset", &ExposureT::subset, "bbox"_a, "origin"_a = PARENT);
+
+            cls.def("writeFits", (void (ExposureT::*)(std::string const &) const) &ExposureT::writeFits);
+            cls.def("writeFits", (void (ExposureT::*)(fits::MemFileManager &) const) &ExposureT::writeFits);
+            cls.def("writeFits", [](ExposureT &self, fits::Fits &fits) { self.writeFits(fits); });
+
+            cls.def("writeFits",
+                    [](ExposureT &self, std::string const &filename, fits::ImageWriteOptions const &imageOptions,
+                       fits::ImageWriteOptions const &maskOptions, fits::ImageWriteOptions const &varianceOptions) {
+                        self.writeFits(filename, imageOptions, maskOptions, varianceOptions);
+                    },
+                    "filename"_a, "imageOptions"_a, "maskOptions"_a, "varianceOptions"_a);
+            cls.def("writeFits",
+                    [](ExposureT &self, fits::MemFileManager &manager, fits::ImageWriteOptions const &imageOptions,
+                       fits::ImageWriteOptions const &maskOptions, fits::ImageWriteOptions const &varianceOptions) {
+                        self.writeFits(manager, imageOptions, maskOptions, varianceOptions);
+                    },
+                    "manager"_a, "imageOptions"_a, "maskOptions"_a, "varianceOptions"_a);
+            cls.def("writeFits",
+                    [](ExposureT &self, fits::Fits &fits, fits::ImageWriteOptions const &imageOptions,
+                       fits::ImageWriteOptions const &maskOptions, fits::ImageWriteOptions const &varianceOptions) {
+                        self.writeFits(fits, imageOptions, maskOptions, varianceOptions);
+                    },
+                    "fits"_a, "imageOptions"_a, "maskOptions"_a, "varianceOptions"_a);
+
+            cls.def_static("readFits", (ExposureT(*)(std::string const &)) ExposureT::readFits);
+            cls.def_static("readFits", (ExposureT(*)(fits::MemFileManager &)) ExposureT::readFits);
+
+            cls.def("getCutout", &ExposureT::getCutout, "center"_a, "size"_a);
+
+        });
 }
 
 PYBIND11_MODULE(exposure, mod) {
-    py::module::import("lsst.afw.image.exposureInfo");
-    py::module::import("lsst.afw.image.maskedImage");
+    lsst::utils::python::WrapperCollection wrappers(mod, "lsst.afw.image.exposure");
+    wrappers.addInheritanceDependency("lsst.afw.image.exposureInfo");
+    wrappers.addInheritanceDependency("lsst.afw.image.maskedImage");
```

Comment:
Are you planning to link the object files for `exposure`, `exposureInfo`, and `maskedImage` into one module on a future ticket?  I'm a bit surprised to see them on this ticket's PR at all, given that the ticket description was for `afw.fits` and `afw.geom`.  (I have essentially the same question about all of the other `afw.image` submodules, but I won't comment on them further, because I assume the answer is the same.)